.github/workflows/ci.yml

```diff
@@ -9,7 +9,9 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.14.7
+          go-version: 1.15.6
+      - name: Create golang symlink
+        run: sudo ln -f -s $GOROOT/bin/* /usr/bin/ 
```

Comment:
why do we need this? I would suggest to avoid `sudo` unless necessary

Reply:
It's because of this issue with Github actions - https://github.com/actions/virtual-environments/issues/731#issuecomment-617039721

Reply:
New setup-go version fixed it. Removed sudo.